internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -989,3 +991,9 @@ func processOCIEnvToParam(envs []string) map[string]string {
 
 	return paramEnv
 }
+
+// isPrivilegedContainerCreationRequest returns if a given container
+// creation request would create a privileged container
+func isPrivilegedContainerCreationRequest(spec *specs.Spec) bool {
+	return spec.Annotations[annotations.LCOWPrivileged] == "true"
```

Comment:
overkill, but can we switch to `"github.com/Microsoft/hcsshim/internal/oci".ParseAnnotationsBool` here to ward against the annotation being `"True"` or some other caps-variation.

Reply:
given this is how it is done elsewhere for this value, i think that should be a separate PR that addresses that.

Reply:
I've opened a PR to address this comment:

https://github.com/microsoft/hcsshim/pull/1628